plutus-chain-index/e2e-tests/sanity-test/spec/from-hash/stake-validator.spec.ts

```diff
@@ -0,0 +1,19 @@
+import * as env from '../../testnet-env.json';
+import { expect } from "chai";
+const request = require("supertest")(env.url);
+
+//there's any example of stake validator onchain yet
```

Comment:
Add a TODO saying that this should be update once there's a stake validator on chain. 

I would comment the test as it's a duplication of another one.